### PR TITLE
Subnautica: compose DeathLink custom text instead of overwriting

### PR DESCRIPTION
**Pull request by Berserker66**
## What is this fixing or adding?
Prevents divergence if core text is updated. (Which it was.)

## How was this tested?
image

### CODE REVIEW

**Review by SunCatMC — APPROVED** [review]
Tested it myself, LGTM

File: worlds/subnautica/options.py [hunk 1]
```diff
@@ -112,8 +112,7 @@ def get_pool(self) -> typing.List[str]:
 
 
 class SubnauticaDeathLink(DeathLink):
-    """When you die, everyone dies. Of course the reverse is true too.
-    Note: can be toggled via in-game console command "deathlink"."""
+    __doc__ = DeathLink.__doc__ + "\n    Note: can be toggled via in-game console command \"deathlink\"."
```
Comment (Exempt-Medic) [inline worlds/subnautica/options.py:115]: Presumably you meant this instead?
```suggestion
    __doc__ = DeathLink.__doc__ + "\n\n    Note: can be toggled via in-game console command \"deathlink\"."
```

Comment (SunCatMC) [reply worlds/subnautica/options.py:115]: original didn't have a line break either

Comment (Berserker66) [reply worlds/subnautica/options.py]: True, but I kind of like the line break.

File: worlds/subnautica/options.py [hunk 2]
```diff
@@ -112,8 +112,7 @@ def get_pool(self) -> typing.List[str]:
 
 
 class SubnauticaDeathLink(DeathLink):
-    """When you die, everyone dies. Of course the reverse is true too.
-    Note: can be toggled via in-game console command "deathlink"."""
+    __doc__ = DeathLink.__doc__ + "\n\n    Note: can be toggled via in-game console command \"deathlink\"."
```
Comment (SunCatMC) [inline worlds/subnautica/options.py:115]:
```suggestion
    __doc__ = DeathLink.__doc__ + "\n" + """
    Note: can be toggled via in-game console command "deathlink".
    """
```
suggestion to how format this so you don't need to escape anything

Comment (Berserker66) [reply worlds/subnautica/options.py:115]: I know, I chose not to.

**Review by Exempt-Medic — APPROVED** [review]
LGTM and displays just fine on the site and in a template

**Review by NewSoupVi — APPROVED** [review]
Sure

**Pull request merged**